azure/scripts/ansible/playbooks/roles/autoscaler/tasks/main.yml

```diff
@@ -0,0 +1,41 @@
+---
+- name: Get Machine Set
+  ansible.builtin.command: |
+    oc get machineset -n openshift-machine-api -o jsonpath='{.items[0].metadata.labels.machine\.openshift\.io/cluster-api-cluster}'
+    --kubeconfig /home/"{{ BOOTSTRAP_ADMIN_USERNAME }}"/.kube/config
```

Comment:
Is it possible to use the native kubernetes module for this instead of `command`? The default location that the kubernetes module and `oc` or `kubectl` check for the config file is `~/.kube/config`. Therefore there should be no reason to use the `--kubeconfig` flag. 

If this is due to the file being under a different user's home directory, consider using `become_user: "{{ BOOTSTRAP_ADMIN_USERNAME }}"`

Reply:
Removed the `--kubeconfig` flag. Working as expected.

Reply:
Unfortunately, I couldn't find the usage of native kubernetes module for the following.
1. oc project
2. oc create secret
3. oc adm policy
4. oc get machineset

Thus, used `ansible.builtin.command` for those.